styles.css

```diff
@@ -0,0 +1,542 @@
+
+* {
+  scrollbar-width: none;
+  list-style: none;
+  box-sizing: border-box;
+}
+
+*::-webkit-scrollbar {
+  display: none;
+}
+
+.check,
+.check * {
+  border: solid 1px rgb(0, 0, 0);
+}
+
+#header {
+  /* position: absolute; */
+  width: 100vw;
+  left: 0;
+  z-index: 10;
+  border-bottom: 2px solid gray;
+  background-color: white;
+}
+
+#header a {
+  color: black;
+  text-decoration: none;
+}
+
+#header a:hover {
+  color: black;
+  text-decoration: none;
+}
+
+#header-nav-bar {
+  display: flex;
+  flex-direction: row;
+  column-gap: 37%;
+}
+
+.session-container {
+  display: none;
+}
+
+.secondary-nav {
+  display: none;
+}
+
+#menu {
+  position: absolute;
+  z-index: 1;
+}
+
+.close-menu-icon {
+  position: absolute;
+  display: none;
+  cursor: pointer;
+
+  /* opacity: 0; */
+}
+
+#menu-container {
+  position: absolute;
+  font-weight: bold;
+  top: 4em !important;
+  width: 100%;
+  height: 100%;
+
+  /* opacity: 0; */
+}
+
+.menu {
+  flex-direction: column;
+  align-items: center;
+  row-gap: 2rem;
+  padding: 1rem 1rem 1rem 1rem;
+  background-color: lightgray;
+  display: none;
+}
+
+.nav-li1:hover {
+  background-color: orangered;
+}
+
+#menu-icon {
+  cursor: pointer;
+  transition: all 0.3s ease-in-out;
+}
+
+#menu-icon:hover #menu-icon:focus {
+  transform: scale(1.1);
+  outline: none;
+}
+
+.hero {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 2rem;
+  margin-top: 1.5rem;
+  width: 100%;
+}
+
+.hero-hello {
```

Comment:
Please follow the guideline, use only color provided on color swatch.
<img width="401" alt="Screenshot 2023-03-18 at 12 13 35 PM" src="https://user-images.githubusercontent.com/25887253/226096646-d2c9895a-42d6-4067-b7b8-e795a8e97fc8.png">

Reply:
Great! thank you! I've changed it!